baybe/kernels/base.py

```diff
@@ -0,0 +1,30 @@
+"""Base classes for all kernels."""
+
+from abc import ABC
+
+from attrs import define
+
+from baybe.serialization.core import (
+    converter,
+    get_base_structure_hook,
+    unstructure_base,
+)
+from baybe.serialization.mixin import SerialMixin
+
+
+@define(frozen=True)
+class Kernel(ABC, SerialMixin):
+    """Abstract base class for all kernels."""
+
+    def to_gpytorch(self, **kwargs):
```

Comment:
why the **kwargs?

Reply:
This ensures that everything that you pass as keyword argument is passed to the constructor of the gpytorch kernel, no matter what exactly you pass. I thought that this was the best/most elegant way of ensuring this "passthrough", if there is a better solution I am more than happy to change this 😄

Reply:
could be both

additionally I wonder where the optional `nu` parameter you can specify int he new baybe `MaternKernel` class comes into play, doesnt seem to be the case?

Reply:
True, I still need to parse `*args*` and also the fields, thanks for spotting!

Reply:
I have attempted to fix this using the `unstructure_base` method on `self` to get all of the fields the corresponding class might have, extracting the fields, and extending `kwargs`.
Is this acceptable? It works, but I do not know if there is a safer/more elegant way.

Reply:
As discussed in the meeting and the chat, I have now changed this and created a corresponding utility function in `baybe.utils.basic`. Please let me know if this solution is now more favorable.

Reply:
Since there was no comment regarding this issue in both of your PRs from yesterday, I'll mark this as resolved. There is also another open comment regarding this function, so in case there is something to discuss, we can do it there.